server/views/referrals/show/risksAndNeeds/partials/layout.njk

```diff
@@ -0,0 +1,42 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
```

Comment:
In an upcoming PR, I've actually replaced this in favour of a rootLayout which can be used by all sub sections under viewing a referral, risks and needs and status history.